kore/src/Kore/Rewrite/SMT/Lemma.hs

```diff
@@ -87,6 +87,7 @@ declareSMTLemmas m = do
             runTranslator $
                 translatePredicateWith SideCondition.top translateUninterpreted $
                     wrapPredicate $ sentenceAxiomPattern axiomDeclaration
+        -- TODO (Andrei B) Should I do the same for the predicates from  TranslatorState?
```

Comment:
If, before, the `Predicate`s were part of the `terms`, then they would have been affected by this code as well. If that's the case, now we should definitely run it on the `predicates` as well.